iot-hub/Samples/device/PnpDeviceSamples/Thermostat/ThermostatSample.cs

```diff
@@ -108,10 +121,10 @@ private async Task GetWritablePropertiesAndHandleChangesAsync()
 
                 foreach (KeyValuePair<string, object> propertyUpdate in twinCollection)
                 {
-                    string componentName = propertyUpdate.Key;
```

Comment:
There is only `propertyName` instead of `componentName` in the context of Thermostat sample.